src/sidebar.js

```diff
@@ -94,13 +89,18 @@ class Sidebar extends React.Component {
     // filter out if a user starts swiping with a second finger
     if (!this.isTouching()) {
       let touch = ev.targetTouches[0];
+      let tcX = touch.clientX;
+      if (this.props.pullRight) {
+        tcX = this.state.sidebarWidth - tcX;
```

Comment:
We shouldn't couple the fact that the sidebar is in pull right mode with the code that handles the touches. The logic should be incorporated in the render method where we're actually deciding on the props and are pull right aware.